lib/openxml/vml/elements/rounded_rectangle.rb

```diff
@@ -0,0 +1,97 @@
+          attribute :border_top_color, expects: :string, displays_as: :bordertopcolor
+          attribute :bullet, expects: :explicit_true_false
+          attribute :button, expects: :explicit_true_false
+          attribute :bw_mode, displays_as: :bwmode, one_of: %i(auto black blackTextAndLines color grayOutline grayScale hide highContrast inverseGray lightGrayscale undrawn white)
+          attribute :bw_normal, displays_as: :bwnormal, one_of: %i(auto black blackTextAndLines color grayOutline grayScale hide highContrast inverseGray lightGrayscale undrawn white)
+          attribute :bw_pure, displays_as: :bwpure, one_of: %i(auto black blackTextAndLines color grayOutline grayScale hide highContrast inverseGray lightGrayscale undrawn white)
+          attribute :clip, expects: :explicit_true_false
+          attribute :clip_to_wrap, expects: :explicit_true_false, displays_as: :cliptowrap
+          attribute :connector_type, one_of: %i(curved elbow none straight), displays_as: :connectortype
+          attribute :diagram_node_layout, expects: :valid_diagram_layout, displays_as: :dgmlayout
+          attribute :diagram_node_recent_layout, expects: :valid_diagram_layout, displays_as: :dgmlayoutmru
+          attribute :diagram_node_kind, expects: :integer, displays_as: :dgmnodekind
+          attribute :double_click_notify, expects: :explicit_true_false, displays_as: :doubleclicknotify
+          attribute :force_dash, expects: :explicit_true_false, displays_as: :forcedash
+          attribute :hr, expects: :explicit_true_false
+          attribute :hr_align, one_of: %i(center left right), displays_as: :hralign
+          attribute :hr_no_shade, expects: :explicit_true_false, displays_as: :hrnoshade
+          attribute :hr_percent, expects: :integer, displays_as: :hrpct
+          attribute :hr_standard, expects: :explicit_true_false, displays_as: :hrstd
+          attribute :inset_mode, one_of: %i(auto custom), displays_as: :insetmode
+          attribute :ole, expects: :explicit_true_false
+          attribute :ole_icon, expects: :explicit_true_false, displays_as: :oleicon
+          attribute :hide_extra_handles, expects: :explicit_true_false, displays_as: :oned
+          attribute :prefer_relative, expects: :explicit_true_false, displays_as: :preferrelative
+          attribute :regroup_id, expects: :integer, displays_as: :regroupid
+          attribute :optional_string, expects: :string, displays_as: :spid
+          attribute :optional_number, expects: :string, displays_as: :spt # Actually expects float
+          attribute :user_drawn, expects: :explicit_true_false, displays_as: :userdrawn
+          attribute :user_hidden, expects: :explicit_true_false, displays_as: :userhidden
+        end
+
+      private
+
+        def valid_arc_size(value)
+          message = "Invalid arc size. Requires a percent or value in terms of f."
+          raise ArgumentError, message unless value.is_a?(String) && value =~ /^[0-9]+[%f]$/
```

Comment:
Future Refactor, apart from this PR.

Could you add `matches: /re/` to `attribute`'s definition the same way as @cacqw7's `one_of: []`?